app/src/main/java/com/oracle/javafx/scenebuilder/app/util/AppSettings.java

```diff
@@ -160,10 +162,10 @@ private static void updateLatestVersionInfo() {
                 latestVersionText = announcementObject.getString("text");
                 latestVersionAnnouncementURL = announcementObject.getString("url");
             } catch (IOException e) {
-                e.printStackTrace();
+                Logger.getLogger(AppSettings.class.getName()).log(Level.WARNING, "Failed to read latest version json: ", e);
             }
         } catch (MalformedURLException e) {
-            e.printStackTrace();
+            Logger.getLogger(AppSettings.class.getName()).log(Level.WARNING, "Failed to construct latest version info URL: ", e);

```

Comment:
Isn't it better to create a single `static final` instance of the Logger for the class?

Reply:
Would only do that if there are multiple calls at various places inside a class.
Would not create that if it is only an exception handling.

Opposite would be for example when logging state of object or node tree modifications, there I'd hold a (`private static final`) logger instance for this particular class.

Reply:
For this class it would be possibly okay to hold a Logger instance. But, I also think this work is important and it's not necessary to be perfect from day one. Actually I'd appreciate if we could get this into Scene Builder 19. It will possibly turn out helpful as we would eventually get some useful logfiles in future.

Also, we could discuss how logging should be handled in future. I offer to run the necessary refactorings to implement the rules which we possibly might agree upon. But as of now, this PR will already be quite useful if merged.